packages/react/spec/useAction.spec.tsx

```diff
@@ -417,6 +417,54 @@ describe("useAction", () => {
     `);
   });
 
+  test("generates correct mutation and variables for fat trigger", async () => {
```

Comment:
nit: since this is open source, may be worth using a more descriptive name (most folks won't know what a "fat" trigger is)

Reply:
"trigger that contributes inputs"